src/components/DarkMode.vue

```diff
@@ -11,23 +11,23 @@
 <script>
 export default {
   name: "Darkmode",
-  data: function () {
-    return {
-      isDark: null,
-    };
+  props: {
+    isDark: Boolean,
   },
   created: function () {
-    this.isDark =
+    let isDark =
       "overrideDark" in localStorage
         ? JSON.parse(localStorage.overrideDark)
-        : matchMedia("(prefers-color-scheme: dark)").matches;
-    this.$emit("updated", this.isDark);
+        : this.isDark === null
+        ? matchMedia("(prefers-color-scheme: dark)").matches
+        : this.isDark;
```

Comment:
Chaining 2 ternary operator is not the best for readability, could you replace it with `if / else` statements?

Reply:
Sure, I was trying to match the syntax of a similar code block already in the project. Not sure where it was but I'll update mine.